packer/sea-starter-kit-win2019-eval.pkr.hcl

```diff
@@ -136,6 +142,59 @@ build {
         "Expand-Archive ./${var.archive_name}.zip -Destination ./${var.archive_name}"
     ]
   }
+  
+  provisioner "comment" {
+    comment     = "Extracting ${var.landing_page}.zip to c:/Ed-Fi"
+    ui          = true
+    bubble_text = false
+  }
+
+  provisioner "powershell" {
+    debug_mode        = "${var.debug_mode}"
+    elevated_password = "${var.user_name}"
+    elevated_user     = "${var.password}"
+    inline            = [
+		"New-Item -ItemType Directory -Path c:/Ed-Fi -Force | Out-Null",
+        "Set-Location c:/temp",
+        "Expand-Archive ./${var.landing_page}.zip -Destination c:/Ed-Fi"
+    ]
+  }
+  
+  provisioner "comment" {
+    comment     = "Create shortcut for landing page"
+    ui          = true
+    bubble_text = false
+  }
+
+  provisioner "powershell" {
+    debug_mode        = "${var.debug_mode}"
+    elevated_password = "${var.user_name}"
+    elevated_user     = "${var.password}"
+    inline            = [
+		"((Get-Content -path C:/Ed-Fi/docs/index.html -Raw) -replace '@@DOMAINNAME@@',[System.Net.Dns]::GetHostName()) | Set-Content -Path C:/Ed-Fi/docs/index.html",
```

Comment:
we should be consistent with the rest of the file and use [Environment]::MachineName

Reply:
updated